docs/G-Codes.md

```diff
@@ -369,6 +369,23 @@ see the [BL-Touch guide](BLTouch.md)):
 - `BLTOUCH_STORE MODE=<output_mode>`: This stores an output mode in the
   EEPROM of a BLTouch V3.1 Available output_modes are: `5V`, `OD`
 
+## Annexed Probe
```

Comment:
Should be Dockable Probe

Reply:
I have made this correction on my local repo already I just have not pushed the commits to github. Thanks for catching it though.